Playgrounds/AudioKitPlaygrounds/Playgrounds/Effects.playground/Pages/Multi-tap Delay.xcplaygroundpage/Contents.swift

```diff
@@ -17,7 +17,7 @@ player.looping = true
 //: that mixes together several delays and gains.
 var delays = [AKVariableDelay]()
 
-func multitapDelay(_ input: AKNode?, times: [Double], gains: [Double]) -> AKMixer {
+func multitapDelay(_ input: AKNode2?, times: [Double], gains: [Double]) -> AKMixer {
```

Comment:
Given 1a0196c, why `AKNode2` here?

Reply:
playgrounds don't get compiled so that was easy to miss. fixed